trimmer.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.slice(0, 10)
```

Comment:
- [ ] As stated in the project requirement in the following screen, kindly consider the name has a maximum of 10 characters. If it's longer it should trim the word. 
```ruby
    def correct_name
      @nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name
    end
```
>![image](https://github.com/newhoteng/OOP-school-library/assets/53402600/8499f6c7-b6d5-46ed-8eba-144638c81f6b)